brainscore_vision/model_helpers/activations/core.py

```diff
@@ -265,15 +439,62 @@ def attach_stimulus_set_meta(assembly, stimulus_set):
     stimulus_paths = [str(stimulus_set.get_stimulus(stimulus_id)) for stimulus_id in stimulus_set['stimulus_id']]
     stimulus_paths = [lstrip_local(path) for path in stimulus_paths]
     assembly_paths = [lstrip_local(path) for path in assembly['stimulus_path'].values]
+
     assert (np.array(assembly_paths) == np.array(stimulus_paths)).all()
-    assembly['stimulus_path'] = stimulus_set['stimulus_id'].values
+    assembly = assembly.reset_index('presentation')
+    assembly.assign_coords(stimulus_path=('presentation', stimulus_set['stimulus_id'].values))
```

Comment:
is this different from
```suggestion
    assembly['stimulus_path'] = 'presentation', stimulus_set['stimulus_id'].values)
```
?

Reply:
Not in this case, but used this to keep it uniform with the multi-coord assignment below